src/table/table.scss

```diff
@@ -59,8 +59,8 @@
       transform: translateX(50%);
       touch-action: none;
       cursor: ew-resize;
-      z-index: 100;
-      display: none;
+      z-index: 1;
+      opacity: 0;
 
```

Comment:
It is a bit strange when your cursor is on right side column but left side column is highlighted. But I think this is minor as a cursor changes.

Reply:
It makes _some_ sense because you are resizing the left column. Any changes to the right column size would just be incidental (due to flex).